### PR TITLE
Fix minor grammatical mistakes in architecture.md 

### PR DESCRIPTION
**Pull request by mrMarce**
Fixed minor grammatical errors

### CODE REVIEW

**Comment by aka-jain** [comment]
@RishabhJain2018 check this please.

File: docs/source/architecture.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ Django is the heart of the application, which powers our backend. We use Django
 
 #### Django Rest Framework
 
-We use Django Rest Framework for writing and providing REST APIs. It's permission and serializers have helped write a maintainable codebase.
+We use Django Rest Framework for writing and providing REST APIs. Its permission and serializers have helped write a maintainable codebase.
 
 #### RabbitMQ
 
-We currently use RabbitMQ for queueing submission messages which are then later on processed by a Python worker.
+We currently use RabbitMQ to queue submissions for processing by a python worker.
```
Comment (RishabhJain2018) [inline docs/source/architecture.md]: @mrMarce Please revert this change or change it such that it conveys the complete meaning.

**Comment by deshraj** [comment]
Closing this due to inactivity.
